onnxsim/onnx_simplifier.py

```diff
@@ -130,10 +131,13 @@ def is_dynamic(node):
     return copy.deepcopy(const_nodes)
 
 
-def forward(model, inputs=None, input_shapes: Optional[TensorShapes] = None) -> Dict[str, np.ndarray]:
+def forward(model, inputs: Optional[dict] = None, input_shapes: Optional[TensorShapes] = None,
```

Comment:
How about creating a `Tensors = Dict[Optional[str], np.array]` in line 16 and replace the  `Optional[dict]` in this line and line 156 with `Optional[Tensors]`?

Reply:
sure, that's great